app/src/main/res/drawable/ic_location_on_blue_24dp.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#2D6AFF"
```

Comment:
format

Reply:
vector phải format à ông, mấy pull trước t tạo mỗi vector k :D

Reply:
format hết đi ông :>>

Reply:
thay cả dimension nữa ông, đa màn hình là toang đấy

Reply:
Tức là mỗi màn hình sẽ có 1 file dimen riêng nhưng làm sao để biết: ví dụ hiện tại chạy trên máy tôi đặt là 180dp chẳng hạn, sang màn hình khác to hơn hoặc nhỏ hơn thì sẽ thành bao nhiêu?
t thử library dimenify nhưng k đc